recipes/cairo/meson/conanfile.py

```diff
@@ -86,21 +86,21 @@ def layout(self):
         basic_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("pixman/0.42.2")
+        self.requires("pixman/0.43.0")
         if self.options.with_zlib and self.options.with_png:
-            self.requires("expat/2.5.0")
+            self.requires("expat/2.6.0")
```

Comment:
```suggestion
            self.requires("expat/2.6.2")
```
I assume you are not using the latest version for some specific reason?

Reply:
Bumping it built fine for me without conflicts.

Reply:
conflicts with fontconfig so reverting this change